src/ol/interaction/Extent.js

```diff
@@ -272,6 +287,16 @@ class Extent extends PointerInteraction {
     return vertexFeature;
```

Comment:
You do not need the `removePointFeature_` function when you work on the return value here: `return mapBrowserEvent.dragging` might do the trick, but I have not tested that.